.github/workflows/enums.yml

```diff
@@ -0,0 +1,103 @@
+name: Custom checks
```

Comment:
```suggestion
name: Check For Enum Changes
```
or a similar name?

Reply:
I've called the Github action 'Custom checks' because the same action file can specify multiple jobs. The job is called 'Detect enum changes' and what you see is: 

![image](https://user-images.githubusercontent.com/107591/100340697-7e63b280-2fd3-11eb-9be8-8508cfc848de.png)

which may be descriptive enough? I was thinking we could add more jobs into 'Custom checks' in the future. Jobs can run in parallel and even depend on other jobs, such as a setup job.

Reply:
Renamed 'Custom checks' to 'Pull request checks', as the overall action is PR-oriented.